script/deploy_dags/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/astronomer/ap-airflow:2.2.4-onbuild
```

Comment:
Can we move this entire scripts folder to `.circleci` please so all the testing code is in a single place